aptos-move/framework/aptos-framework/sources/aggregator/README.md

```diff
@@ -0,0 +1,47 @@
+# Design Rationale
```

Comment:
May be this can be moved into dev docs? cc @clay-aptos

Reply:
> May be this can be moved into dev docs? cc @clay-aptos

Yes, all of these docs should be considered for Aptos.dev. See my issue for this in:
https://github.com/aptos-labs/aptos-core/issues/5577

Just ping that issue when you are ready to work on it. We have myriad docs in the repo that are not linked from Aptos.dev, it seems.

Reply:
Sounds good! Maybe this can wait until the blogpost first then.